src/typechat.ts

```diff
@@ -111,7 +114,8 @@ export function createJsonTranslator<T extends object>(model: TypeChatLanguageMo
             if (!attemptRepair) {
                 return error(`JSON validation failed: ${validation.message}\n${jsonText}`);
             }
-            prompt += `${responseText}\n${typeChat.createRepairPrompt(validation.message)}`;
```

Comment:
We'll be sending the responseText twice. Once as {assistant, responseText} and also as part of the prompt.